api/src/main/java/com/epam/pipeline/manager/search/SearchRequestBuilder.java

```diff
@@ -258,6 +259,7 @@ private QueryBuilder prepareFacetedQuery(final String requestQuery) {
 
     private void addTermAggregationToSource(final SearchSourceBuilder searchSource, final String facet) {
         final TermsAggregationBuilder aggregationBuilder = AggregationBuilders.terms(facet)
+                .size(MAX_AGGREGATIONS_SIZE)
```

Comment:
Could we move this setting to system preferences?

Reply:
done